process_report/process_report.py

```diff
@@ -30,6 +29,11 @@
     bu_subsidy_processor,
     prepayment_processor,
 )
+from process_report.invoices.invoice import (
```

Comment:
All of our other files import using `from process_report.invoices import invoice`. For uniformity, please use that pattern instead.

Reply:
You also did not include [`INVOICE_DATE_FIELD`](https://github.com/CCI-MOC/invoicing/blob/0a8b8623072b0cb55022b5ed1053fa002eb124f0/process_report/process_report.py#L440). This would have raised an error if the top module was ran.